### PR TITLE
[main] update zero one lists

### PR DESCRIPTION
**Pull request by hi-8**

### CODE REVIEW

File: cup/src/pages/zero-one/organisms/gameItem/gameItem.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { motion } from 'framer-motion'
+import styles from '../../styles.module.css'
```
Comment (lamb-sni) [inline cup/src/pages/zero-one/organisms/gameItem/gameItem.tsx]: [must]
Don't use "../".
Please replace "../../styles.module.css" -> "@/pages/zero-one/styles.module.css"

File: cup/src/pages/zero-one/organisms/gameList/gameList.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { motion } from 'framer-motion'
+import styles from '../../styles.module.css'
```
Comment (lamb-sni) [inline cup/src/pages/zero-one/organisms/gameList/gameList.tsx]: same as above

File: cup/src/pages/zero-one/organisms/gameList/gameList.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import { ZERO_ONE_TYPES } from '@/constants/zeroOne'
+import GameItem from '../gameItem/gameItem'
```
Comment (lamb-sni) [inline cup/src/pages/zero-one/organisms/gameList/gameList.tsx]: same as above

**Review by lamb-sni — CHANGES_REQUESTED** [review]
Hi, Hi-8 !
Thx for first contribution to our project.

I have some Request changes.
Please check it out.

**Comment by lamb-sni** [comment]
hey hey hey

**Comment by hi-8** [comment]
test comment

**Comment by lamb-sni** [comment]
yo yo yo

File: cup/src/pages/zero-one/organisms/gameItem/gameItem.tsx (hunk 1 above)
Comment (hi-8) [reply cup/src/pages/zero-one/organisms/gameItem/gameItem.tsx]: @lamb-sni 
thx. I have made the necessary corrections.

**Review by lamb-sni — APPROVED** [review]
VERY GOOD
LGTM

**Pull request merged**